README.md

```diff
@@ -23,10 +35,78 @@ The monorepo is structured into four packages:
 - @t3n/icons
 - @t3n/configs
 
+## Setup
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+This is an example of how to list things and their versions you need to use the software and how to install them.
+
+```sh
+
+npm install npm@latest -g
+
+```
+
+### Installation
+
+1. Clone the repo
+
+```sh
+git clone git@github.com:t3n/react-packages.git
+```
+
+2. Install NPM packages
+
+```sh
+npm install
+```
+
+3. Build packages
+
+```sh
+npm run build
+```
+
+**Run this command from the main package**. This will run the build script of each package in the correct order, as **@t3n/storybook** depends on **@t3n/components**, which itself depends on **@t3n/theme** and **@t3n/icons**.
+
 ## Development
 
-First, install all dependencies by running `npm install`.
+How to run and develop in this project.
+
+```sh
+npm run start:storybook
+```
+
+This will start storybook and handle local bundling of packages in the monorepo to enable the quickest development workflow possible. You will then be able to develop components in storybook.
+
+:warning:
+
+This will however **NOT** create bundles for all packages, so before committing changes, you should **ALWAYS** run `npm run build` to build all packages and prepare for releasing
```

Comment:
Bundles werden sowieso nicht mit ins Repo gepushed, das passiert in der Pipeline. `npm run build` sollte vor dem committen ausgeführt werden, damit die Types generiert werden.

Reply:
Ich seh grad, das stand vorher auch bereits so in der Readme, könnte man aber eben nochmal abändern.

Reply:
ok, mach ich!